### PR TITLE
🐛 Use iam_contracts for IIdentity

### PR DESCRIPTION
**Pull request by alexanderkasten**
This PR uses the correct Identity interface.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@process-engine/solutionexplorer.repository.processengine_datastore",
-  "version": "1.0.0",
+  "name": "@prcess-engine/solutionexplorer.repository.processengine_datastore",
```
Comment (robpal94) [inline package.json]: Typo I guess 😉

File: src/SolutionExplorerProcessEngineRepositoryDatastore.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import {IIdentity, IQueryClause} from '@essential-projects/core_contracts';
+import {IQueryClause} from '@essential-projects/core_contracts';
```
Comment (ElRaptorus) [inline src/SolutionExplorerProcessEngineRepositoryDatastore.ts]: Is there any way we can get rid of this fast and easy? 
Would be nice if we can kick out core_contracts altogether.

Comment (cmg-dev) [reply src/SolutionExplorerProcessEngineRepositoryDatastore.ts]: What is this used for.

Comment (alexanderkasten) [reply src/SolutionExplorerProcessEngineRepositoryDatastore.ts]:
```
   const query: IQueryClause = {
       attribute: 'name',
       operator: '=',
       value: diagramName,
     };
```

@cmg-dev only for that

Comment (alexanderkasten) [reply src/SolutionExplorerProcessEngineRepositoryDatastore.ts]: @Vyperus Can we delete this repository?

**Pull request merged**